include/ttmlir/Dialect/TTNN/IR/TTNNOps.td

```diff
@@ -567,8 +567,8 @@ def TTNN_EmbeddingOp : TTNN_NamedDPSOp<"embedding"> {
     }];
 
     let arguments = (ins AnyRankedTensor:$input,
```

Comment:
Does this change affect some tests?

Reply:
Nope, every embedding test we have right now targets only TTIR dialect.